src/Game.js

```diff
@@ -0,0 +1,53 @@
+
+class Game {
+
+  constructor() {
+    this.rolls = [];
+    this.currentScore = 0;
+  }
+
+  roll(pins) {
+  this.rolls.push(pins);
+  }
+
+  score() {
+  let score = 0;
+  let rollIndex = 0;
+
+  for(let frameIndex = 0; frameIndex < 10; frameIndex++) {
+    if (this.isStrike(rollIndex)) {
+      score += this.strikeBonus(rollIndex);
+      rollIndex++;
+      continue;
+    }
+
+    const frameScore = this.rolls[rollIndex] + this.rolls[rollIndex + 1];
+
```

Comment:
it appears that you've written the logic for strikes and spares, but you haven't really tested all these functions below that you've written (yet). They're tidy and readable though.